Source/Core/VideoCommon/TextureDecoder_Common.cpp

```diff
@@ -617,6 +635,24 @@ void TexDecoder_DecodeTexel(u8* dst, const u8* src, int s, int t, int imageWidth
     *((u32*)dst) = color;
   }
   break;
+  case TextureFormat::XFB:
+  {
+    // TODO: I should kind of like... ACTUALLY TEST THIS!!!!!
```

Comment:
This is for software right?  I admittedly haven't tested software yet, that's why I left it in.